MonoXAnalysis/python/analyzers/dmCore_modules_cff.py

```diff
@@ -464,7 +474,7 @@ def doKalmanMuonCorrections(sync=False,smear="basic"):
     lheWeightAna,
     skimAnalyzer,
    #eventSelector,
-    jsonAna,
+    ###jsonAna,
```

Comment:
it was just a test to run directly on a specific file of a MC dataset. I've tried to use dmCoreSequence.remove(jsonAna), but seemed not to work in my case.